sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/src/autorest.md

```diff
@@ -481,4 +481,8 @@ directive:
     where: $.paths['/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/backupFabrics/{fabricName}/protectionContainers/{containerName}']
     transform: >
       $.put['x-ms-long-running-operation'] = true;
+  - from: bms.json
+    where: $.paths['/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/backupFabrics/{fabricName}/protectionContainers/{containerName}/protectedItems/{protectedItemName}']
+    transform: >
+      $.put['x-ms-long-running-operation'] = true;
```

Comment:
Can you help to rise a S360 ticket for service team.

Reply:
https://dev.azure.com/devdiv/DevDiv/_workitems/edit/1915949